File: DSA/large.cpp
#include <iostream>
using namespace std;
// Code to find largest no. in array
int main()
{
    int n, i, largest = 0;
    cin >> n;
    int arr[n];
    cout << "Enter numbers ? ";
    for (i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    for (i = 0; i < n; i++)
    {
        if (arr[i] > largest)
        {
            largest = arr[i];
        }
    }
    cout << " " << largest;
    return 0;
}

File: DSA/large.java
import java.util.Scanner;

public class large {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int n, i, largest = 0;
            n = scan.nextInt();
            int[] arr = new int[n];
            System.out.print("Enter numbers");
            for (i = 0; i < n; i++) {
                arr[i] = scan.nextInt();
            }
            for (i = 0; i < n; i++) {
                if (arr[i] > largest) {
                    largest = arr[i];
                }
            }
            System.out.print(" " + largest);
            scan.close();
        }
    }
}
